jmclient/jmclient/wallet_utils.py

```diff
@@ -332,7 +334,8 @@ def get_imported_privkey_branch(wallet, m, showprivkey):
         addr = wallet.get_addr_path(path)
         script = wallet.get_script_path(path)
         balance = 0.0
-        for data in wallet.get_utxos_by_mixdepth_()[m].values():
+        for data in wallet.get_utxos_by_mixdepth_(
+            include_disabled=True)[m].values():
```

Comment:
confusing indent. My linter would suggest to indent this line one more level so it's not confused with the loop's content.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/334/commits/20879bbda047131676cb4d41992469a1a1a04d60